wagtailcontact/forms.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from django.conf import settings
+from django.core.mail import EmailMessage
+from django.template import loader
+
+
+class ContactForm(forms.Form):
+
+    subject_template = "wagtailcontact/email/subject.txt"
+    txt_template = "wagtailcontact/email/message.txt"
+    success_url = None
+
+    name = forms.CharField(max_length=100, required=True)
+    email = forms.EmailField(max_length=200, required=True)
+    message = forms.CharField(
+        widget=forms.Textarea(attrs={'rows': 4, 'cols': False}),
+        required=True,
+    )
+
+    def get_email_context(self):
+        ctx = self.cleaned_data
+        ctx['subject_prefix'] = settings.EMAIL_SUBJECT_PREFIX
+        return ctx
+
+    def send_email(self, to, subject_template, txt_template, reply_to):
+        context = self.get_email_context()
+        from_email = settings.DEFAULT_FROM_EMAIL
+        subject = loader.render_to_string(subject_template, context)
+        message = loader.render_to_string(txt_template, context)
+        msg = EmailMessage(
+            subject.strip(),
+            message,
+            from_email,
+            to,
+            reply_to=reply_to
+        )
+        msg.send()
+
+    def get_to(self, page):
+        try:
+            return [page.enquiry_email]
```

Comment:
This mixin needs to provide an `enquiry_email` field.

Reply:
This seems interesting, adding the field requires us to add it to Page.content_panels(). Doing this without inheriting `Page` fails.
@solarissmoke will it be ok to have a `ContactPage` which inherits from this mixin and `Page`, which can then be used by subclasses?

Reply:
The problem with subclassing a `Page` model is that the model is not abstract, and you will end up with multiple table joins just to fetch a single page object.

You should not need to have to subclass page, however. Just set `content_panels` = `Page.content_panels + [...]`. Without subclassing.

It will mean you have to add Wagtail to your dependencies (maybe don't pin a version though).